data/json/recipes/recipe_deconstruction.json

```diff
@@ -3601,6 +3811,24 @@
     "components": [ [ [ "cardboard", 2 ] ], [ [ "plastic_chunk", 2 ] ] ],
     "flags": [ "BLIND_EASY" ]
   },
+  {
+    "result": "pan",
+    "type": "uncraft",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "2 m",
+    "qualities": [ { "id": "SAW_M", "level": 1 } ],
+    "components": [ [ [ "sheet_metal_small", 8 ], [ "steel_chunk", 1 ] ] ]
```

Comment:
This only has 50ml of space for metal. Much like the other items, volume should be increased to bring it in line.

Reply:
Sure, will increase it's volume.